deploy-agent/deployd/common/utils.py

```diff
@@ -231,9 +232,45 @@ def redeploy_check(labels, service, redeploy) -> int:
         return redeploy + 1
     return 0
     
+def redeploy_check_non_container(commit, service, redeploy, healthcheckConfigs):
+    if "HEALTHCHECK_REDEPLOY_WHEN_UNHEALTHY" in healthcheckConfigs and healthcheckConfigs["HEALTHCHECK_REDEPLOY_WHEN_UNHEALTHY"] == "True":
+        log.info(f"Auto redeployment is enabled on service {service}")
+        if "HEALTHCHECK_REDEPLOY_MAX_RETRY" in healthcheckConfigs and redeploy < int(healthcheckConfigs["HEALTHCHECK_REDEPLOY_MAX_RETRY"]) or \
+            "HEALTHCHECK_REDEPLOY_MAX_RETRY" not in healthcheckConfigs and redeploy < 3:
```

Comment:
is 3 a default value? probably add a const some where for the default max retry redeploy?